pinot-segment-local/src/main/java/org/apache/pinot/segment/local/aggregator/SumValueAggregator.java

```diff
@@ -37,11 +37,17 @@ public DataType getAggregatedValueType() {
 
   @Override
   public Double getInitialAggregatedValue(Number rawValue) {
+    if (rawValue == null) {
```

Comment:
Can this ever be `null`?

Reply:
It could be if the field is null on some of the incoming stream messages. You could always guard against this by filtering out the messages during ingestion, but I think returning 0.0 here as the default should be harmless since this is the sum aggregator

Reply:
Currently the input should never be `null` (it should have already been filled with default value). My concern is that we are adding `null` handling to only this aggregation but not others. In order to completely support `null` input, we need to allow null value in, and annotate the input value as `@Nullable` and support it for all aggregations. That is not in the scope of this PR, so suggest doing it separately